WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -324,8 +335,39 @@ public void loggedInViaUsernamePassword(ArrayList<Integer> oldSitesIds) {
     }
 
     @Override
-    public void help() {
-        ToastUtils.showToast(this, "Help is not implemented yet.");
+    public void helpEmailScreen(String email) {
+        launchHelpshift(null, email,
+                HelpshiftHelper.chooseHelpshiftLoginTag(getLoginMode() == LoginMode.JETPACK_STATS, true));
```

Comment:
`chooseHelpshiftLoginTag` seems to only return wpcom, Jetpack and self-hosted. For when we are not sure which tag is the correct one, do you think it might make sense to add a 4th tag called "Unknown" or something?